modelscan/tools/picklescanner.py

```diff
@@ -219,6 +242,15 @@ def scan_pytorch(
     magic = get_magic_number(data)
     if magic != MAGIC_NUMBER:
         return ScanResults(
-            [], [ModelScanError("pytorch", f"Invalid magic number for file {source}")]
+            [],
+            [
+                ModelScanError(
```

Comment:
Thinking this should be a skipped reason instead of an error. In the benchmarking we're encountering .bin files that aren't pytorch and we should just be skipping them instead of erroring

Reply:
Thank you @swashko for sharing this! I have now updated it to add them to skipped files